app/src/main/java/com/duckduckgo/app/global/DuckDuckGoApplication.kt

```diff
@@ -184,8 +191,7 @@ open class DuckDuckGoApplication : HasActivityInjector, HasServiceInjector, HasS
     }
 
     private fun configureUncaughtExceptionHandler() {
-        val originalHandler = Thread.getDefaultUncaughtExceptionHandler()
-        Thread.setDefaultUncaughtExceptionHandler(AlertingUncaughtExceptionHandler(originalHandler, offlinePixelDataStore))
+        Thread.setDefaultUncaughtExceptionHandler(alertingUncaughtExceptionHandler)
```

Comment:
I'm now creating `AlertingUncaughtExceptionHandler` using Dagger, since it's used in 2 places. Verify this doesn't affect any existing behaviour.